packages/firebase_vertexai/firebase_vertexai/lib/src/live_api.dart

```diff
@@ -107,17 +89,14 @@ final class LiveGenerationConfig extends BaseGenerationConfig {
   /// The speech configuration.
   final SpeechConfig? speechConfig;
 
-  /// The list of desired response modalities.
-  final List<ResponseModalities>? responseModalities;
+  // /// The list of desired response modalities.
+  // final List<ResponseModalities>? responseModalities;
```

Comment:
Should these be removed instead of commented out?

Reply:
removed.